tests/pytest_tests/conftest.py

```diff
@@ -249,3 +249,19 @@ def mock_run_fn(use_magic_mock=False, **kwargs: Any) -> "wandb.sdk.wandb_run.Run
 
     yield mock_run_fn
     unset_globals()
+
+
+@pytest.fixture
+def example_file(tmp_path: Path) -> Path:
+    new_file = tmp_path / "test.txt"
+    new_file.write_text("hello")
+    return new_file
+
+
+@pytest.fixture
+def example_files(tmp_path: Path) -> Path:
```

Comment:
this seems only relevant to artifact tests? maybe doesn't belong at top level conftest?

Reply:
It's kind of nice because we use it in both systems_test and in unit_test, but I can duplicate it to those two if you prefer.

Reply:
yeah sorry i was referring to the second fixture! This one is actually nice addition!